lib/OpenQA/Command.pm

```diff
@@ -19,6 +19,7 @@ my $PARAM_RE = qr/^([[:alnum:]_\[\]\.\:]+)=(.*)$/s;
 has apibase => '/api/v1';
 has [qw(apikey apisecret host)];
 has host => 'http://localhost';
+has options => undef;
```

Comment:
I would just assign the defaults here, and use `my $options = $self->options;` in the beginning of the `run` method.

Reply:
Also avoids the `$self->options(\%options)` later in the `run` method.

Reply:
Just make sure it's `has options => sub {{pretty => 0, quiet => 0, links => 0, verbose => 0}};` with a wrapper function, or all instances of the class would share the same hashref, modifying each other's values. Which could get a bit annoying in tests. 😁